tools/mozdevice/logging.js

```diff
@@ -107,6 +107,15 @@ Logging.prototype.error = function(tag, message) {
  */
 Logging.prototype.clear = function() {
   debug('Clearing');
+  if (this.device.properties['ro.build.version.sdk'] >= 21) {
```

Comment:
I prefer to have the workaround code in a if clause, so we can simply remove it later when the issues are addressed. How do you think?